fseai/vision.py

```diff
@@ -0,0 +1,129 @@
+"""
+Computer Vision Packages
+"""
+import torch
+from torchvision import transforms, datasets
+from torch.utils.data import DataLoader
+import torch.nn as nn
+from torch import optim
+from FSEai import fseai_models as models
+
+
+def input_mnist():
+    """
+    Function to load mnist dataset
+    """
+    trainset = datasets.MNIST('', download=True, train=True, transform=transforms.ToTensor())
+    testset = datasets.MNIST('', download=True, train=False, transform=transforms.ToTensor())
+
+    print('MNIST train and test data set Loaded!')
+    return trainset, testset
+
+
+def build_mnist_model():
+    """
+    Function to build a fully connected model 
+    :return: model
```

Comment:
while it's quite popular to use `napoleon` style for docstrings, I found numpy style easy to use